hardhat.config.ts

```diff
@@ -45,6 +45,10 @@ const config: HardhatUserConfig = {
       optimisticEthereum: process.env.OPTIMISM_ETHERSCAN_API_KEY || "",
       optimisticKovan: process.env.OPTIMISM_ETHERSCAN_API_KEY || "",
 
+      // arbitrum
+      arbitrumOne: process.env.ARBISCAN_API_KEY || "",
+      arbitrumGoerli: process.env.ARBISCAN_API_KEY || "",
```

Comment:
Not sure if this will actually work for contract verification since arbiscan still only shows Rinkeby not Goerli.

Reply:
Will be nice when everyone stops supporting Rinkeby, we can revisit after that.